src/crimson/osd/osd_operations/common/pg_subop_blocker.h

```diff
@@ -0,0 +1,53 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include "osd/osd_op_util.h"
+#include "crimson/osd/osd_operation.h"
+
+namespace crimson::osd {
+
+using interruptor =
+  ::crimson::interruptible::interruptor<
+    ::crimson::osd::IOInterruptCondition>;
+
+// bases on 998cb8c141bb89aafae298a9d5e130fbd78fe5f2
+template <typename T>
+struct SubOpBlocker : crimson::BlockerT<SubOpBlocker<T>> {
```

Comment:
I don't actually see anything osd/pg specific here -- should the file be subop_blocker.h?

Reply:
Moved to common and renamed